BeVolt/Drivers/src/LTC6811.c

```diff
@@ -75,7 +75,185 @@ Copyright 2017 Linear Technology Corp. (LTC)
 /*********************************************************/
 /*** Code that was added by UTSVT. ***/
 /*********************************************************/
-void LTC6811_Init(cell_asic *battMod){	
+
+static void cs_set(uint8_t state){
+	if(state == 0){
+		GPIO_ResetBits(GPIOB, GPIO_Pin_6);
+	}else{
+		GPIO_SetBits(GPIOB, GPIO_Pin_6);
+	}
+}
+void delay_u(uint16_t micro){
```

Comment:
@SijWoo More of a general comment, but we should agree on either spaces or tabs for indent so that we don't have weird formatting issues.